doc/_scripts/software_maturity/software_maturity_features.yaml

```diff
@@ -22,10 +22,27 @@ features:
     Sidewalk over LORA: SIDEWALK && SIDEWALK_LINK_MASK_LORA
     Sidewalk - OTA DFU over Bluetooth LE: SIDEWALK && SIDEWALK_DFU_SERVICE_BLE
   bluetooth:
-    Bluetooth LE Peripheral/Central: BT_CTLR && (BT_CENTRAL || BT_PERIPHERAL)
-    LLPM: BT_CTLR_SDC_LLPM
+    LE 2M PHY: BT_CTLR_PHY_2M_SUPPORT
+    Peripheral: BT_CTLR && BT_PERIPHERAL
+    Central: BT_CTLR && BT_CENTRAL
+    Data Length Extensions: BT_CTLR_DATA_LEN_UPDATE_SUPPORT
+    Advertising Extensions: BT_CTLR_ADV_EXT_SUPPORT
+    Periodic Advertising: BT_CTLR_ADV_PERIODIC_SUPPORT
```

Comment:
I would assume we need to use `BT_CTLR_ADV_PERIODIC` instead of `BT_CTLR_ADV_PERIODIC_SUPPORT`. The latter is only a dummy symbol used to check if the feature can be enabled at all

Reply:
should I remove the `SUPPORT` suffix for other features as well?

Reply:
I would assume so. I would recommend having a look at how they are defined here: https://github.com/nrfconnect/sdk-zephyr/blob/main/subsys/bluetooth/controller/Kconfig